### PR TITLE
add billable checkbox to utilization panel

### PR DESCRIPTION
**Pull request by tkellen**

### CODE REVIEW

File: src/client/modules/components/review-employee-row-day/template.html [hunk 1]
```diff
@@ -64,6 +65,11 @@
       <option value="{{this}}">{{name}}</option>
       {{/each}}
     </select>
+    {{billable}}
```
Comment (jugglinmike) [inline src/client/modules/components/review-employee-row-day/template.html]: Can you remove this?

Comment (tkellen) [inline src/client/modules/components/review-employee-row-day/template.html]: Yup!

Comment (tkellen) [inline src/client/modules/components/review-employee-row-day/template.html]: Done.

**Comment by tkellen** [comment]
Thanks @jugglinmike!

**Pull request merged**